### ISSUE TITLE
[Accessibility]: First round of accessibility issues

### ISSUE THREAD

**Issue by AshleyHenry15** [posit-dev/product-doc-theme#55]
**Describe the accessibility issue**
A clear and concise description of what the accessibility is.

Lighthouse report surfaced the following issues:

Can't fix:
- [ ] Search button in nav bar: Inaccessible button name. Quarto issue: https://github.com/quarto-dev/quarto-cli/issues/678

Can fix:
- [ ] Footer logos (posit guide and posit logo): Lacking alt text
- [ ] Nav bar text: contrast between font color and background (contrast is too low)

**Resolution**
If applicable, add the solution to resolve this accessibility issue.

**Additional context**
